mars/services/task/execution/ray/executor.py

```diff
@@ -426,7 +448,9 @@ async def get_available_band_resources(self) -> Dict[BandType, Resource]:
             idx = 0
             for band_resource in band_resources:
                 for band, resource in band_resource.items():
-                    virtual_band_resources[(f"ray_virtual://{idx}", band)] = resource
+                    virtual_band_resources[
+                        (f"ray_virtual_address_{idx}:0", band)
```

Comment:
The band name of mars on ray is like this: `ray://mars_cluster_1653388173/0/0`, should we keep it be consistent with that?

Reply:
The virtual worker address is for compatible with Mars worker ip address.